etc/schema.json

```diff
@@ -3733,7 +3735,8 @@
                     }
                 },
                 "memcap_pressure": {
-                    "description": "Percentage of memcaps used by flow, stream, stream-reassembly and app-layer-http",
+                    "description":
+                            "Percentage of memcaps used by flow, stream, stream-reassembly and app-layer-http",
```

Comment:
nit: clang formatting changes unrelated to schema additions?

Reply:
indeed. have done this so many times separately in the recent PRs 😩 got lazy. Thanks, will fix